### ISSUE TITLE
Discriminator loss and Generator loss is weird?

### ISSUE THREAD

**Issue by PeterKim1** [bfarzin/pytorch_aae#5]
Hi. Thanks for your awesome codes.

I'm studying AAE, so I try to run your codes.

But when i see Discriminator loss and Generator loss, i think shape is weird.

![image](https://user-images.githubusercontent.com/57930520/111736263-d5fa9680-88c0-11eb-9a2a-7a3403d3ffbd.png)

(I can't use tensorflow logger in your code, so I make another tensorboard. But same value is used.)

Is it okay??
